modules/graceful-shutdown.adoc

```diff
@@ -61,6 +62,15 @@ Shutting down the nodes using one of these methods allows pods to terminate grac
 +
 [NOTE]
```

Comment:
This is more of a suggestion, but I would consider revising these two notes as a separate section. Having two notes back to back lessens their visibility and impact on the user. If all of this information is important, maybe pulling it out of this section and emphasizing it with a header is better?

Reply:
Agree. I plan to open a new PR to track this change.

Reply:
As mentioned earlier, I created a new [PR](https://github.com/openshift/openshift-docs/pull/42334). I made the step in the note optional, instead of creating a separate section to avoid back-to-back notes.